src/runners/helpers/dbconfig.py

```diff
@@ -15,18 +15,26 @@ def random_string(n=10, alphabet=ascii_uppercase):
 else:
     tail = ''
 
+
+
 # database & account properties
-REGION = environ.get('REGION', "us-west-2")
+# For customer: HOST = '', PORT = '443', PROTOCOL = 'https'
+# For internal use: REGION = ''
+HOST = environ.get('SNOWFLAKE_HOST', '') # '' for customer,
+PORT = environ.get('SH_PORT')
+PROTOCOL = environ.get('SH_PROTOCOL')
+
+REGION = environ.get('SH_REGION', "us-west-2")
 REGION_SUBDOMAIN_POSTFIX = '' if REGION == 'us-west-2' else f'.{REGION}'
-ACCOUNT = environ.get('SNOWFLAKE_ACCOUNT', '') + REGION_SUBDOMAIN_POSTFIX
+ACCOUNT = environ.get('SH_ACCOUNT', '') if HOST else environ.get('SH_ACCOUNT', '') + REGION_SUBDOMAIN_POSTFIX
```

Comment:
keep this env var name the same, as it's the Snowflake account used for SnowAlert